router/transformer/transformer.go

```diff
@@ -150,11 +156,52 @@ func (trans *HandleT) Transform(transformType string, transformMessage *types.Tr
 	return destinationJobs
 }
 
+func (trans *HandleT) ResponseTransform(responseData integrations.DeliveryResponseT, destName string) (statusCode int, respBody string) {
+	rawJSON, err := json.Marshal(responseData)
+	if err != nil {
+		panic(err)
+	}
+	var resp *http.Response
+	var respData []byte
+	url := getResponseTransformURL(destName)
+	trans.transformerResponseTransformRequestTime.Start()
+	resp, err = trans.client.Post(url, "application/json; charset=utf-8", bytes.NewBuffer(rawJSON))
+	if err != nil {
+		respData = []byte("")
+		trans.logger.Errorf("[Transformer Response Transform request] :: %+v", err)
+		return http.StatusInternalServerError, string(respData)
+	}
+	trans.transformerResponseTransformRequestTime.End()
+	if resp != nil && resp.Body != nil {
+		respData, _ = io.ReadAll(resp.Body)
+	}
+	integrations.CollectIntgErrorStats([]byte(gjson.GetBytes(respData, "output").Raw), false)
+	var contentTypeHeader string
+	if resp != nil && resp.Header != nil {
+		contentTypeHeader = resp.Header.Get("Content-Type")
+	}
+	if contentTypeHeader == "" {
+		//Detecting content type of the respBody
+		contentTypeHeader = http.DetectContentType(respData)
```

Comment:
why do we need lines 180-184 when we can get the content type using the detectcontenttype function

Reply:
Yeah we can remove that logic